{\rtf1\ansi\ansicpg1252\cocoartf2578
\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fnil\fcharset0 Menlo-Regular;\f1\fnil\fcharset0 Menlo-Bold;}
{\colortbl;\red255\green255\blue255;\red0\green0\blue0;\red56\green185\blue199;\red57\green192\blue38;
}
{\*\expandedcolortbl;;\csgray\c0;\cssrgb\c25544\c77008\c82022;\cssrgb\c25704\c77963\c19556;
}
\margl1440\margr1440\vieww11520\viewh8400\viewkind0
\pard\tx560\tx1120\tx1680\tx2240\tx2800\tx3360\tx3920\tx4480\tx5040\tx5600\tx6160\tx6720\pardirnatural\partightenfactor0

\f0\fs22 \cf2 \CocoaLigature0 (djangoPy3Env) macbookpro@MacBookPros-MacBook-Pro dojo_ninja_proj % python3 manage.py shell \
Python 3.9.5 (v3.9.5:0a7dcbdb13, May  3 2021, 13:17:02) \
[Clang 6.0 (clang-600.0.57)] on darwin\
Type "help", "copyright", "credits" or "license" for more information.\
(InteractiveConsole)\
>>> dojo.objects.creat(name = "nasser" , city = "nablus",state = "palestine")\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
NameError: name 'dojo' is not defined\
>>> Dojo.objects.creat(name = "nasser" , city = "nablus",state = "palestine")\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
NameError: name 'Dojo' is not defined\
>>> Dojo.objects.create(name = "nasser" , city = "nablus",state = "palestine")\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
NameError: name 'Dojo' is not defined\
>>> exit()\
(djangoPy3Env) macbookpro@MacBookPros-MacBook-Pro dojo_ninja_proj % python3 manage.py makemigrations\
No changes detected\
(djangoPy3Env) macbookpro@MacBookPros-MacBook-Pro dojo_ninja_proj % python3 manage.py migrate\

\f1\b \cf3 Operations to perform:
\f0\b0 \cf2 \

\f1\b   Apply all migrations: 
\f0\b0 admin, auth, contenttypes, dojo_ninjas_app, sessions\

\f1\b \cf3 Running migrations:
\f0\b0 \cf2 \
  No migrations to apply.\
(djangoPy3Env) macbookpro@MacBookPros-MacBook-Pro dojo_ninja_proj % python3 manage.py shell         \
Python 3.9.5 (v3.9.5:0a7dcbdb13, May  3 2021, 13:17:02) \
[Clang 6.0 (clang-600.0.57)] on darwin\
Type "help", "copyright", "credits" or "license" for more information.\
(InteractiveConsole)\
>>> Dojo.objects.create(name="nasser",city="nablus",state="palestine")\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
NameError: name 'Dojo' is not defined\
>>> import models from Dojo\
  File "<console>", line 1\
    import models from Dojo\
                  ^\
SyntaxError: invalid syntax\
>>> from dojo_ninjas_app.models import*\
>>> Dojo.objects.create(name="nasser",city="nablus",state="palestine")\
<Dojo: Dojo object (1)>\
>>> Dojo.objects.create(name="ahmadr",city="ramallah",state="palestine")\
<Dojo: Dojo object (2)>\
>>> Dojo.objects.create(name="rami",city="jenin",state="palestine")\
<Dojo: Dojo object (3)>\
>>> c=Dojo.objects.get()\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/manager.py", line 82, in manager_method\
    return getattr(self.get_queryset(), name)(*args, **kwargs)\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/query.py", line 410, in get\
    raise self.model.MultipleObjectsReturned(\
dojo_ninjas_app.models.Dojo.MultipleObjectsReturned: get() returned more than one Dojo -- it returned 3!\
>>> c.delete()\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
NameError: name 'c' is not defined\
>>> c= Dojo.objects.get(id=1)\
>>> c.delete()\
(1, \{'dojo_ninjas_app.Ninjas': 0, 'dojo_ninjas_app.Dojo': 1\})\
>>> c= Dojo.objects.get(id=0)\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/manager.py", line 82, in manager_method\
    return getattr(self.get_queryset(), name)(*args, **kwargs)\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/query.py", line 406, in get\
    raise self.model.DoesNotExist(\
dojo_ninjas_app.models.Dojo.DoesNotExist: Dojo matching query does not exist.\
>>> c= Dojo.objects.get(id=1)\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/manager.py", line 82, in manager_method\
    return getattr(self.get_queryset(), name)(*args, **kwargs)\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/query.py", line 406, in get\
    raise self.model.DoesNotExist(\
dojo_ninjas_app.models.Dojo.DoesNotExist: Dojo matching query does not exist.\
>>> c= Dojo.objects.get(id=2)\
>>> c.delete()\
(1, \{'dojo_ninjas_app.Ninjas': 0, 'dojo_ninjas_app.Dojo': 1\})\
>>> c= Dojo.objects.get(id=3)\
>>> c.delete()\
(1, \{'dojo_ninjas_app.Ninjas': 0, 'dojo_ninjas_app.Dojo': 1\})\
>>> Dojo.objects.create(name="rami",city="jenin",state="palestine")\
<Dojo: Dojo object (4)>\
>>> Dojo.objects.create(name="ahmad",city="nablus",state="palestine")\
<Dojo: Dojo object (5)>\
>>> Dojo.objects.create(name="ahmad",city="jericho",state="palestine")\
<Dojo: Dojo object (6)>\
>>> this_dojo = Dojo.objects.get(id=4)\
>>> my_ninja = Ninja.objects.create(first_name="nasser",last_name="Sayeh",dojo=this_dojo)\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
NameError: name 'Ninja' is not defined\
>>> my_ninja = Ninjas.objects.create(first_name="nasser",last_name="Sayeh",dojo=this_dojo)\
>>> my_ninja = Ninjas.objects.create(first_name="maryam",last_name="ibrahim",dojo=this_dojo)\
>>> my_ninja = Ninjas.objects.create(first_name="hani",last_name="khmais",dojo=this_dojo)\
>>> this_dojo = Dojo.objects.get(id=5)\
>>> my_ninja = Ninja.objects.create(first_name="ahmad",last_name="Sayeh",dojo=this_dojo)\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
NameError: name 'Ninja' is not defined\
>>> my_ninja = Ninjas.objects.create(first_name="ahmad",last_name="Sayeh",dojo=this_dojo)\
>>> my_ninja = Ninjas.objects.create(first_name="hani",last_name="samer",dojo=this_dojo)\
>>> my_ninja = Ninjas.objects.create(first_name="rahaf",last_name="hosari",dojo=this_dojo)\
>>> this_dojo = Dojo.objects.get(id=6)\
>>> my_ninja = Ninjas.objects.create(first_name="rahaf",last_name="hosari",dojo=this_dojo)\
>>> my_ninja = Ninjas.objects.create(first_name="nasser",last_name="sayeh",dojo=this_dojo)\
>>> my_ninja = Ninjas.objects.create(first_name="mohammad",last_name="hosanini",dojo=this_dojo)\
>>> Dojo.objects.first\
<bound method BaseManager._get_queryset_methods.<locals>.create_method.<locals>.manager_method of <django.db.models.manager.Manager object at 0x7fd0414e0310>>\
>>> Dojo.objects.first.get()\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
AttributeError: 'function' object has no attribute 'get'\
>>> Dojo.objects.first.all()\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
AttributeError: 'function' object has no attribute 'all'\
>>> Dojo.objects.first()\
<Dojo: Dojo object (4)>\
>>> Ninjas.objects.first()\
<Ninjas: Ninjas object (1)>\
>>> dojo = Dojo.objects.get(id=4)\
>>> dojo.ninja\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
AttributeError: 'Dojo' object has no attribute 'ninja'\
>>> Ninja.objects.get(id = dojo)\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
NameError: name 'Ninja' is not defined\
>>> Ninjas.objects.get(id = dojo)\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/manager.py", line 82, in manager_method\
    return getattr(self.get_queryset(), name)(*args, **kwargs)\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/query.py", line 399, in get\
    clone = self.filter(*args, **kwargs)\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/query.py", line 892, in filter\
    return self._filter_or_exclude(False, *args, **kwargs)\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/query.py", line 910, in _filter_or_exclude\
    clone.query.add_q(Q(*args, **kwargs))\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1290, in add_q\
    clause, _ = self._add_q(q_object, self.used_aliases)\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1315, in _add_q\
    child_clause, needed_inner = self.build_filter(\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1251, in build_filter\
    condition = self.build_lookup(lookups, col, value)\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1116, in build_lookup\
    lookup = lookup_class(lhs, rhs)\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/lookups.py", line 20, in __init__\
    self.rhs = self.get_prep_lookup()\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/lookups.py", line 70, in get_prep_lookup\
    return self.lhs.output_field.get_prep_value(self.rhs)\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 966, in get_prep_value\
    return int(value)\
TypeError: int() argument must be a string, a bytes-like object or a number, not 'Dojo'\
>>> Ninjas.objects.get(id = 1)\
<Ninjas: Ninjas object (1)>\
>>> Ninjas.objects.all()\
<QuerySet [<Ninjas: Ninjas object (1)>, <Ninjas: Ninjas object (2)>, <Ninjas: Ninjas object (3)>, <Ninjas: Ninjas object (4)>, <Ninjas: Ninjas object (5)>, <Ninjas: Ninjas object (6)>, <Ninjas: Ninjas object (7)>, <Ninjas: Ninjas object (8)>, <Ninjas: Ninjas object (9)>]>\
>>> Ninjas.objects.filter(dojos=dojo)\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/manager.py", line 82, in manager_method\
    return getattr(self.get_queryset(), name)(*args, **kwargs)\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/query.py", line 892, in filter\
    return self._filter_or_exclude(False, *args, **kwargs)\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/query.py", line 910, in _filter_or_exclude\
    clone.query.add_q(Q(*args, **kwargs))\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1290, in add_q\
    clause, _ = self._add_q(q_object, self.used_aliases)\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1315, in _add_q\
    child_clause, needed_inner = self.build_filter(\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1190, in build_filter\
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1049, in solve_lookup_type\
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())\
  File "/Users/macbookpro/Desktop/python_stack/djangoPy3Env/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1419, in names_to_path\
    raise FieldError("Cannot resolve keyword '%s' into field. "\
django.core.exceptions.FieldError: Cannot resolve keyword 'dojos' into field. Choices are: dojo, dojo_id, first_name, id, last_name\
>>> Ninjas.objects.filter(dojo=dojo)\
<QuerySet [<Ninjas: Ninjas object (1)>, <Ninjas: Ninjas object (2)>, <Ninjas: Ninjas object (3)>]>\
>>> dojo = Dojo.objects.last()\
>>> Ninjas.objects.filter(dojo=dojo)\
<QuerySet [<Ninjas: Ninjas object (7)>, <Ninjas: Ninjas object (8)>, <Ninjas: Ninjas object (9)>]>\
>>> Ninjas.objects.last()\
<Ninjas: Ninjas object (9)>\
>>> exit()\
(djangoPy3Env) macbookpro@MacBookPros-MacBook-Pro dojo_ninja_proj % python3 manage.py makemigrations\
You are trying to add a non-nullable field 'desc' to dojo without a default; we can't do that (the database needs something to populate existing rows).\
Please select a fix:\
 1) Provide a one-off default now (will be set on all existing rows with a null value for this column)\
 2) Quit, and let me add a default in models.py\
Select an option: 1\
Please enter the default value now, as valid Python\
The datetime and django.utils.timezone modules are available, so you can do e.g. timezone.now\
Type 'exit' to exit this prompt\
>>> "nasser sayeh"\

\f1\b \cf3 Migrations for 'dojo_ninjas_app':
\f0\b0 \cf2 \
  
\f1\b dojo_ninjas_app/migrations/0002_dojo_desc.py
\f0\b0 \
    - Add field desc to dojo\
(djangoPy3Env) macbookpro@MacBookPros-MacBook-Pro dojo_ninja_proj % python3 manage.py shell\
Python 3.9.5 (v3.9.5:0a7dcbdb13, May  3 2021, 13:17:02) \
[Clang 6.0 (clang-600.0.57)] on darwin\
Type "help", "copyright", "credits" or "license" for more information.\
(InteractiveConsole)\
>>> from dojos_ninja_app import*\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
ModuleNotFoundError: No module named 'dojos_ninja_app'\
>>> from dojo_ninjas_app import*\
>>> Dojo.objects.create(name = "nasser",city="nablus"state="palestine",desc="im 29 years old")\
  File "<console>", line 1\
    Dojo.objects.create(name = "nasser",city="nablus"state="palestine",desc="im 29 years old")\
                                                     ^\
SyntaxError: invalid syntax\
>>> Dojo.objects.create(name = "nasser",city="nablus",state="palestine",desc="im 29 years old")\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
NameError: name 'Dojo' is not defined\
>>> exit()\
(djangoPy3Env) macbookpro@MacBookPros-MacBook-Pro dojo_ninja_proj % python3 manage.py migrate       \

\f1\b \cf3 Operations to perform:
\f0\b0 \cf2 \

\f1\b   Apply all migrations: 
\f0\b0 admin, auth, contenttypes, dojo_ninjas_app, sessions\

\f1\b \cf3 Running migrations:
\f0\b0 \cf2 \
  Applying dojo_ninjas_app.0002_dojo_desc...
\f1\b \cf4  OK
\f0\b0 \cf2 \
(djangoPy3Env) macbookpro@MacBookPros-MacBook-Pro dojo_ninja_proj % python3 manage.py shell\
Python 3.9.5 (v3.9.5:0a7dcbdb13, May  3 2021, 13:17:02) \
[Clang 6.0 (clang-600.0.57)] on darwin\
Type "help", "copyright", "credits" or "license" for more information.\
(InteractiveConsole)\
>>> from dojo_ninjas_app import*\
>>> Dojo.objects.create(name="nasser",city="nablus",state="palestine",desc="im 29 years old")\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
NameError: name 'Dojo' is not defined\
>>> from dojo_ninjas_app import Dojo\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
ImportError: cannot import name 'Dojo' from 'dojo_ninjas_app' (/Users/macbookpro/Desktop/python_stack/Python_stack/django/django_into/dojo_ninjas_shell/dojo_ninja_proj/dojo_ninjas_app/__init__.py)\
>>> from dojo_ninjas_app import*\
>>> Dojo.objects.create(name="nasser",city="nablus",state="palestine",desc="im 29 years old")\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
NameError: name 'Dojo' is not defined\
>>> exit()\
(djangoPy3Env) macbookpro@MacBookPros-MacBook-Pro dojo_ninja_proj % python3 manage.py makemigrations\
No changes detected\
(djangoPy3Env) macbookpro@MacBookPros-MacBook-Pro dojo_ninja_proj % python3 manage.py makemigrations\

\f1\b \cf3 Migrations for 'dojo_ninjas_app':
\f0\b0 \cf2 \
  
\f1\b dojo_ninjas_app/migrations/0003_remove_dojo_desc.py
\f0\b0 \
    - Remove field desc from dojo\
(djangoPy3Env) macbookpro@MacBookPros-MacBook-Pro dojo_ninja_proj % python3 manage.py shell         \
Python 3.9.5 (v3.9.5:0a7dcbdb13, May  3 2021, 13:17:02) \
[Clang 6.0 (clang-600.0.57)] on darwin\
Type "help", "copyright", "credits" or "license" for more information.\
(InteractiveConsole)\
>>> from dojo_ninjas_app import Dojo\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
ImportError: cannot import name 'Dojo' from 'dojo_ninjas_app' (/Users/macbookpro/Desktop/python_stack/Python_stack/django/django_into/dojo_ninjas_shell/dojo_ninja_proj/dojo_ninjas_app/__init__.py)\
>>> exit()\
(djangoPy3Env) macbookpro@MacBookPros-MacBook-Pro dojo_ninja_proj % python3 manage.py makemigrations\
No changes detected\
(djangoPy3Env) macbookpro@MacBookPros-MacBook-Pro dojo_ninja_proj % python3 manage.py migrate       \

\f1\b \cf3 Operations to perform:
\f0\b0 \cf2 \

\f1\b   Apply all migrations: 
\f0\b0 admin, auth, contenttypes, dojo_ninjas_app, sessions\

\f1\b \cf3 Running migrations:
\f0\b0 \cf2 \
  Applying dojo_ninjas_app.0003_remove_dojo_desc...
\f1\b \cf4  OK
\f0\b0 \cf2 \
(djangoPy3Env) macbookpro@MacBookPros-MacBook-Pro dojo_ninja_proj % python3 manage.py shell         \
Python 3.9.5 (v3.9.5:0a7dcbdb13, May  3 2021, 13:17:02) \
[Clang 6.0 (clang-600.0.57)] on darwin\
Type "help", "copyright", "credits" or "license" for more information.\
(InteractiveConsole)\
>>> from dojo_ninjas_app import Dojo\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
ImportError: cannot import name 'Dojo' from 'dojo_ninjas_app' (/Users/macbookpro/Desktop/python_stack/Python_stack/django/django_into/dojo_ninjas_shell/dojo_ninja_proj/dojo_ninjas_app/__init__.py)\
>>> from dojo_ninjas_app import Ninjas\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
ImportError: cannot import name 'Ninjas' from 'dojo_ninjas_app' (/Users/macbookpro/Desktop/python_stack/Python_stack/django/django_into/dojo_ninjas_shell/dojo_ninja_proj/dojo_ninjas_app/__init__.py)\
>>> exit()\
(djangoPy3Env) macbookpro@MacBookPros-MacBook-Pro dojo_ninja_proj % python3 manage.py makemigrations\
You are trying to add a non-nullable field 'desc' to dojo without a default; we can't do that (the database needs something to populate existing rows).\
Please select a fix:\
 1) Provide a one-off default now (will be set on all existing rows with a null value for this column)\
 2) Quit, and let me add a default in models.py\
Select an option: 1\
Please enter the default value now, as valid Python\
The datetime and django.utils.timezone modules are available, so you can do e.g. timezone.now\
Type 'exit' to exit this prompt\
>>> \
Please enter some code, or 'exit' (with no quotes) to exit.\
>>> \
Please enter some code, or 'exit' (with no quotes) to exit.\
>>> ""\

\f1\b \cf3 Migrations for 'dojo_ninjas_app':
\f0\b0 \cf2 \
  
\f1\b dojo_ninjas_app/migrations/0004_dojo_desc.py
\f0\b0 \
    - Add field desc to dojo\
(djangoPy3Env) macbookpro@MacBookPros-MacBook-Pro dojo_ninja_proj % python3 manage.py migrate\

\f1\b \cf3 Operations to perform:
\f0\b0 \cf2 \

\f1\b   Apply all migrations: 
\f0\b0 admin, auth, contenttypes, dojo_ninjas_app, sessions\

\f1\b \cf3 Running migrations:
\f0\b0 \cf2 \
  Applying dojo_ninjas_app.0004_dojo_desc...
\f1\b \cf4  OK
\f0\b0 \cf2 \
(djangoPy3Env) macbookpro@MacBookPros-MacBook-Pro dojo_ninja_proj % python3 manage.py shell         \
Python 3.9.5 (v3.9.5:0a7dcbdb13, May  3 2021, 13:17:02) \
[Clang 6.0 (clang-600.0.57)] on darwin\
Type "help", "copyright", "credits" or "license" for more information.\
(InteractiveConsole)\
>>> from dojo_ninjas_app import Dojo\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
ImportError: cannot import name 'Dojo' from 'dojo_ninjas_app' (/Users/macbookpro/Desktop/python_stack/Python_stack/django/django_into/dojo_ninjas_shell/dojo_ninja_proj/dojo_ninjas_app/__init__.py)\
>>> from dojo_ninjas_app import *\
>>> models.objects.all()\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
AttributeError: module 'dojo_ninjas_app.models' has no attribute 'objects'\
>>> model.objects.all()\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
NameError: name 'model' is not defined\
>>> models.object.all()\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
AttributeError: module 'dojo_ninjas_app.models' has no attribute 'object'\
>>> models.all()\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
AttributeError: module 'dojo_ninjas_app.models' has no attribute 'all'\
>>> dojo_ninjas_app.objects.all()\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
NameError: name 'dojo_ninjas_app' is not defined\
>>> Dojo.objects.all()\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
NameError: name 'Dojo' is not defined\
>>> from dojo_ninjas_app import Dojo\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
ImportError: cannot import name 'Dojo' from 'dojo_ninjas_app' (/Users/macbookpro/Desktop/python_stack/Python_stack/django/django_into/dojo_ninjas_shell/dojo_ninja_proj/dojo_ninjas_app/__init__.py)\
>>> from dojo_ninjas_app import Dojo\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
ImportError: cannot import name 'Dojo' from 'dojo_ninjas_app' (/Users/macbookpro/Desktop/python_stack/Python_stack/django/django_into/dojo_ninjas_shell/dojo_ninja_proj/dojo_ninjas_app/__init__.py)\
>>> from dojo_ninjas_app import Dojo\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
ImportError: cannot import name 'Dojo' from 'dojo_ninjas_app' (/Users/macbookpro/Desktop/python_stack/Python_stack/django/django_into/dojo_ninjas_shell/dojo_ninja_proj/dojo_ninjas_app/__init__.py)\
>>> from models import dojo\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
ModuleNotFoundError: No module named 'models'\
>>> from models import Dojo\
Traceback (most recent call last):\
  File "<console>", line 1, in <module>\
ModuleNotFoundError: No module named 'models'\
>>> from dojo_ninjas_app.models import Dojo\
>>> from dojo_ninjas_app.models import*\
>>> Dojo.objects.create(name="nasser",city="nablus",state="palestine".desc="29 years old")\
  File "<console>", line 1\
    Dojo.objects.create(name="nasser",city="nablus",state="palestine".desc="29 years old")\
                                                                          ^\
SyntaxError: invalid syntax\
>>> Dojo.objects.create(name="nasser",city="nablus",state="palestine".desc=29 years old)\
  File "<console>", line 1\
    Dojo.objects.create(name="nasser",city="nablus",state="palestine".desc=29 years old)\
                                                                          ^\
SyntaxError: invalid syntax\
>>> Dojo.objects.create(name="nasser",city="nablus",state="palestine",desc="29 years old")\
<Dojo: Dojo object (7)>}